### PR TITLE
replace panel in toolbar by buttons

### PR DESCRIPTION
**Pull request by jdenisgiguere**
As proposed. Thanks to @fvanderbiest .

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -470,22 +470,7 @@ GEOR.Addons.RVA = Ext.extend(GEOR.Addons.Base, {
     },
 
     createToolbarComponent: function() {
```
Comment (fvanderbiest) [inline js/main.js:472]: now `createToolbarComponents` ?

**Comment by jdenisgiguere** [comment]
There is a regression with this PR. I'm working on it.
Please don't merge.

**Comment by jdenisgiguere** [comment]
Regression fixed by b2ac26e.

File: js/main.js (hunk 1 above)
Comment (jdenisgiguere) [inline js/main.js:472]: We return to a single container.

**Pull request merged**